### ISSUE TITLE
pam_systemd uses different formats for tty in logind's CreateSession call

### ISSUE THREAD

**Issue by Vogtinator** [systemd/systemd#6516]
### Submission type

 - Bug report

### systemd version the issue has been seen with

systemd 234

### Used distribution

openSUSE Tumbleweed 20170730

### Expected behaviour you didn't see

I start both Wayland and X11 sessions (with sddm, but that should not matter here), then run loginctl.
The X11 session is on VT7, the Wayland session on VT2, so I expect:

```
   SESSION        UID USER             SEAT             TTY             
       364       1000 fabian           seat0            tty2
       360       1000 fabian           seat0            tty7
```

### Unexpected behaviour you saw

However, the wayland session is prefixed with `/dev/`.

```
   SESSION        UID USER             SEAT             TTY             
       364       1000 fabian           seat0            /dev/tty2
       360       1000 fabian           seat0            tty7
```

This is because `PAM_TTY` contains the **full path** to the device if it is a device file, according to man 3 pam_set_item:

```
       PAM_TTY
           The terminal name: prefixed by /dev/ if it is a device file; for graphical, X-based, applications the value for this item should be the $DISPLAY variable.
```

and pam_systemd does not account for that.

**Comment by poettering** [comment]
ouch

**Comment by poettering** [comment]
fix waiting in #6582. Please give it a whirl!

**Comment by Vogtinator** [comment]
Applied to v234, shows the correct tty now! I didn't check anything the other changes touch, but didn't notice any breakages either.
